style.css

```diff
@@ -1,5 +1,190 @@
+* {
+    -webkit-box-sizing:border-box;
+    box-sizing:border-box;
+
+}
+
 html {
     font-size: 62.5%;
+    margin: 0;
+}
+
+body {
+    margin: auto;
+    font-family: 'Inconsolata', monospace;
+    font-size: 14px;
+}
+
+a {
+    color: #000;
+    padding: 4px
+}
+
+a:link {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+header {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    background: #eaeaea;
+    padding: 4px;
+    border: solid #000 1px;
+}
+
+@media only screen and (max-width: 600px) {
+    header {
+        display: -webkit-box;
+        display: -ms-flexbox;
+        display: flex;
+        -webkit-box-pack: center;
+        -ms-flex-pack: center;
+        justify-content: center;
+        background: #eaeaea;
+        padding: 4px;
+        border: solid #000 1px;
+    }
+}
+
+.main-nav {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+}
+
+.mini-profile {
+    width: 100%;
+    text-align: center;
+    border: solid #000 1px;
+}
+
+.project-holder {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    text-align: center;
+    list-style: none;
+}
+
+.project {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-flex: 1;
+    flex-shrink: 1;
+    -ms-flex-negative: 1;
+    flex-grow: 1;
+    -ms-flex-positive: 1;
+    flex-basis: 50%;
+    -ms-flex-preferred-size: 50%;
+    color: #fff;
+    background-image: url("https://placekitten.com/700/300");
+    background-size: cover;
+    min-height: 300px;
+    background-position: center;
+    position: relative;
+    padding: 0 100px 0 100px;
 }
 
+@media only screen and (max-width: 600px) {
```

Comment:
You already have a media query for max-width: 600px - see line 43

Reply:
Consolidated media queries and placed them at the end of the doc.